tricity/tricity/spiders/trojmiasto.py

```diff
@@ -1,20 +1,41 @@
 import scrapy
 from tricity.items import HouseItem
+import re
+from datetime import datetime 
+
 
 class HousesspiderSpider(scrapy.Spider):
     name = 'tricity'
     allowed_domains = ['ogloszenia.trojmiasto.pl']
     start_urls = ['https://ogloszenia.trojmiasto.pl/nieruchomosci/dom/gdynia/']
 
+    
+    
     def parse(self, response):
-        houses = response.xpath("//div[@id = 'ogl__list__wrap']").get()
-
+        # Parse each houses 
+        houses = response.css('#ogl__list__wrap .list__item')
+        
         for house in houses:
-            house_item = HouseItem()
-            house_item['price'] = response.xpath("//p[@class = 'list__item__price__value']//text()").get()
-            house_item['price_per_meter'] = response.css('p.list__item__details__info.details--info--price::text').get()
-            house_item['url'] = response.css('a.list__item__content__title__name.link').attrib['href']
-            house_item['area'] = response.xpath("//p[@class='list__item__details__icons__element__desc']//text()").extract_first()
-            house_item['number_of_rooms'] = response.xpath('//ul/li[2]/div/p[2]//text()').extract_first()
+            item = HouseItem()
+            
+            try:
+                item['date'] = datetime.today()
```

Comment:
Nie możesz przypisać żadnej wartości do `item['date'] ` bo w klasie HouseItem nie masz zdefiniowanej property 'date'. Ta linijka rzuca wyjątek i przerywa dalsze scrapowanie i przez to puste obiekty są wpluwane do bazy danych

Reply:
ok